Cluster/Shared Log writes throttled.md

```diff
@@ -0,0 +1,55 @@
+# Shared Log Writes Throttled
+
+## Symptoms
+
+- Stateful Service is experiencing high write transaction latencies.
+- Stateful Service is receiving high count of Timeout Exceptions for write transactions.
+- 'SharedLogWriteThrottled' platform event is raised for the node experiencing the issue. This event is visible on the Events tab of SFX:
+
+![SharedLogWriteThrottled Event](../media/SharedLogWriteThrottled.png)
+
+
+## Possible Causes
+
+Reliable Collections use Write Ahead Log to persist transactions for a Stateful Service. The write ahead log is comprised of the highly performant shared log and the replica dedicated log. The shared log is shared by all the replicas on the node. All transaction records are first written to this shared log. Later, in the background, these records are extracted and flushed to the respective dedicated replica logs.
+Sometimes the shared log may reach its near full capacity (90%). When this happens, the writes are throttled to allow the background task to flush the records to the dedicated log. During this time, the write performance is degraded due to throttling.
+This may occur due to any of the following:
+1.	**Occasional burst in the write rate.** Thus, the rate at which the background task is flushing the records to the dedicated log lags behind the rate at which the shared log is getting filled. This condition is usually transient and will clear up once the write rate is stabilized.
+2.	**Write to the dedicated log failed.** The shared log is implemented as a circular buffer. So, if a dedicated log write fails, the record may block the flushing of the other records which may eventually lead to the shared log full.
+3.	**Replica was aborted.** When the replica is aborted, sometimes the log doesn’t get the chance to flush the records to the replica dedicated log. Thus, the records for the replica remains in the shared log, which may block the shared log truncation and eventually lead to the shared log full.
```

Comment:
If this is a known bug in previous versions, Can we add in the description the version number when bug was fixed ?

Reply:
This is not referring to any bug. Can you please clarify what exactly are you referring to?

Reply:
>When the replica is aborted, sometimes the log doesn’t get the chance to flush the records to the replica dedicated log.

Replica abort is a common scenario. If shared log is becoming full, because of replica abort, it is a bug in our logger. Isn't it ?
I thought we had found those bugs and fixed it in recent versions. Is that not correct ?